seed/filtersets.py

```diff
@@ -88,7 +91,7 @@ def year_filter(self, queryset, name, value):
         name = "{} Calendar Year".format(value)
         cycles = queryset.filter(name__contains=name)
         if not cycles:
-            start = datetime(int(value), 1, 1)
+            start = make_aware(datetime(int(value), 1, 1), pytz.UTC)
```

Comment:
it wouldn't work before because we have timezones enabled. So this was failing. We have to be intentional with timezones all the time now.